src/test/kotlin/three/consulting/epoc/controller/TimesheetControllerIntegrationTest.kt

```diff
@@ -81,4 +82,10 @@ class TimesheetControllerIntegrationTest : ControllerIntegrationTest() {
         )
         assertThat(response.statusCode).isEqualTo(HttpStatus.METHOD_NOT_ALLOWED)
     }
+
+    @Test
+    fun `query for project id 1 returns 200`() {
+        val response = restTemplate.getForEntity("/timesheet?projectId=1", ArrayNode::class.java)
+        assertThat(response.statusCode).isEqualTo(HttpStatus.OK)
+    }
```

Comment:
What about a test for when you try to query timesheets without the provided request parameter?